thermostat-card/README.md

```diff
@@ -0,0 +1,21 @@
+# Thermostat Card
+
+A simple thermostat implemented in CSS based on https://codepen.io/dalhundal/pen/KpabZB
+
+**Options**
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| entity | string | **Required** | `climate.ecobee`
+| min | number | 60 | Minimum value for graph
```

Comment:
defaults should be based on users's set measurements. I mean should depend on fahrenheit vs celsius